broadcast.php

```diff
@@ -374,4 +437,44 @@ function getBroadcastResponseConfirmed($communications_id, &$numbers, &$error)
 	return true;
 }
 
+/**
+* Load tags and the count of numbers for each
+*
+* ...
+* 
+* @param array &$broadcast_tags
+*   Set to an array - keys are the tags, values are the count for each tag
+* @param string &$error
+*   An error if one occurred.
+*   
+* @return bool
+*   True unless an error occurred.
+*/
+
+function loadBroadcastTags(&$broadcast_tags, &$error)
+{
+	$sql = "SELECT DISTINCT tag FROM broadcast_tags ORDER BY tag";
+	if (!db_db_getcol($sql, $tags, $error)) {
```

Comment:
Again, this may just be my confusion, but is $tags initialized? Did you mean $broadcast_tags (since the latter was passed into the function)?

Reply:
Here I am hesitant to initialize every variable that gets retrieved from the database.  The database functions initialize it.